src/robot2017/src/drive_node.cpp

```diff
@@ -41,31 +66,47 @@ int main(int argc, char **argv)
             ROS_WARN_STREAM("Starting in autonomy mode");
         }
     }
+    
+    status.robotCodeActive = true;
+    status.autonomyActive = autoEnable;
+    status.deadmanPressed = false;
 
     RobotExec exec(onPC, debug, autoEnable);
 
-    // establish this program as an ROS node.
-    ros::NodeHandle nh;
-
-    ros::Subscriber sub_tele = nh.subscribe("/robot/teleop", 1000, &RobotExec::teleopReceived, &exec);
-
-    ros::Subscriber sub_aut = nh.subscribe("/robot/autonomy", 1000, &RobotExec::autonomyReceived, &exec);
-
```

Comment:
You also removed the subscriptions for teleop and autonomy - should add them back